packages/modules/src/modules/epics/ContributionsEpic.stories.tsx

```diff
@@ -45,8 +45,11 @@ export const WithBackgroundImage = Template.bind({});
 WithBackgroundImage.args = {
     variant: {
         ...props.variant,
-        backgroundImageUrl:
-            'https://images.unsplash.com/photo-1494256997604-768d1f608cac?ixlib=rb-1.2.1&auto=format&fit=crop&w=1701&q=80',
+        image: {
+            mainUrl:
+                'https://images.unsplash.com/photo-1494256997604-768d1f608cac?ixlib=rb-1.2.1&auto=format&fit=crop&w=1701&q=80',
```

Comment:
Would it be a more useful test if we sourced an image from the Grid, rather than relying on a 3rd party server?

Reply:
hah - I don't know where this originally came from! That's a fair point, I'll swap it for something in-house